cerberus-web/src/test/java/com/nike/cerberus/service/AuthenticationServiceTest.java

```diff
@@ -266,7 +266,7 @@ public void test_that_getKeyId_only_validates_kms_policy_one_time_within_interva
     String accountId = "0000000000";
     String roleName = "role/path";
     String principalArn = String.format("arn:aws:iam::%s:instance-profile/%s", accountId, roleName);
-    String roleArn = String.format(AWS_IAM_ROLE_ARN_TEMPLATE, accountId, roleName);
+    String roleArn = String.format(AWS_IAM_ROLE_ARN_TEMPLATE, "aws", accountId, roleName);
```

Comment:
I'm debating on it. The constant hurts the readability of the regex imo if we decide to replace all occurrences of the partition name. But maybe we can just use constant outside of regex. Thoughts?

Reply:
I'm a pretty big fan of consistency, since we use variables for accountId and roleName I think it makes sense to do the same for the partition. I agree about the readability of the regex but I don't think it makes it any worse than what is already there.